src/v2/models/AppState.js

```diff
@@ -141,8 +141,10 @@ export default Model.extend({
       // make sure to update the correct poll view's refresh value
       const currentFormName = this.get('currentFormName');
       const currentViewState = transformedResponse.remediations.filter(r => r.name === currentFormName)[0];
+      const refreshInterval = currentViewState.refresh ||
+        transformedResponse.currentAuthenticatorEnrollment?.poll?.refresh;
```

Comment:
there might be case when refresh exists at `transformedResponse.currentAuthenticator?.poll?.refresh`?

Reply:
can you add a comment what this is about ? and why a specific authenticator(view) is being used here

Reply:
Instead of here, can we tackle this somewhere specific to AuthenticatorEnrollment ? or in `BaseFormWithPolling`

Reply:
@haishengwu-okta I don't believe I ever see `refresh` in `transformedResponse.currentAuthenticator?.poll?.refresh` but @thomasmaslen-okta if you don't mind confirming.

@pradeepdewda-okta I can add some comment regarding the variables. But I think this should remain here in `AppState.js` since the logic is already here with the `AppState.js` change. And `BaseFormWithPolling` just react to the change in state.

Reply:
@worldleelavivat-okta The issue is we don't want to add View specific code AppState.. If it's not required then just remove it

Reply:
> @haishengwu-okta I don't believe I ever see `refresh` in `transformedResponse.currentAuthenticator?.poll?.refresh` but @thomasmaslen-okta if you don't mind confirming.

(Per Slack conversation) you're right that, due to autoenrollment of the Email Authenticator, you may not see `transformedResponse.currentAuthenticator?.poll?.refresh` in the wild at present -- but, per Haisheng's comment, this PR should definitely anticipate it.

Reply:
@haishengwu-okta / @thomasmaslen-okta  added `currentAuthenticator`

@pradeepdewda-okta 
Added comment + I think this is the cleanest way to get it since this is how remidiations are currently getting the new value as well and we are reusing `BaseFormWithPolling`. 
The problem is that there is no way for the view to get new value since it isn't being re-render + AppState aren't being refresh. Unless we pass on the whole new `transformedResponse` down to the variable instead of just a number to the view then let the view handle extracting data but I feel like that a little heavy handed approach.